pkg/sql/logictest/testdata/logic_test/crdb_internal

```diff
@@ -277,11 +277,15 @@ SELECT * FROM crdb_internal.ranges_no_leases WHERE range_id < 0
 ----
 range_id  start_key  start_pretty  end_key  end_pretty  table_id  database_name  schema_name  table_name  index_name  replicas  replica_localities  learner_replicas  split_enforced_until
 
+statement ok
+CREATE SCHEMA schem; CREATE TABLE schem.bar (y INT PRIMARY KEY)
```

Comment:
nit: i'm curious, whilst you're here, for completeness can you name the schema `"schema"`?

Reply:
done